users/config.rst

```diff
@@ -612,37 +698,61 @@ address
             <address>dynamic</address>
         </device>
 
-paused
+.. option:: device.paused
+
     True if synchronization with this devices is (temporarily) suspended.
 
-allowedNetwork
-    If given, this restricts connections to this device to only this network
-    (see :ref:`allowed-networks`).
+.. option:: device.allowedNetwork
+
+    If given, this restricts connections to this device to only this network.
+    The mechanism is described in detail in a :doc:`separate chapter
+    </advanced/device-allowednetworks>`).
+
+.. option:: device.autoAcceptFolders
+
+    If ``true``, folders shared from this remote device are automatically added
+    and synced locally under the :opt:`default path <defaults.folder>`.  For the
+    folder name, Syncthing tries to use the label from the remote device, and if
+    the same label already exists, it then tries to use the folder's ID.  If
+    that exists as well, it just logs an error.  A local folder already added
+    with the same ID will just be shared rather than created separately.
+
+.. option:: device.maxSendKbps
 
-maxSendKbps
     Maximum send rate to use for this device. Unit is kibibytes/second, despite
     the config name looking like kilobits/second.
 
-maxRecvKbps
+.. option:: device.maxRecvKbps
+
     Maximum receive rate to use for this device. Unit is kibibytes/second,
     despite the config name looking like kilobits/second.
 
-ignoredFolder
+.. option:: device.ignoredFolder
+
     Contains the ID of the folder that should be ignored. This folder will
     always be skipped when advertised from the containing remote device,
     i.e. this will be logged, but there will be no dialog shown in the web GUI.
 
-maxRequestKiB
+.. option:: device.maxRequestKiB
+
     Maximum amount of data to have outstanding in requests towards this device.
     Unit is kibibytes.
 
-remoteGUIPort
+.. option:: device.remoteGUIPort
+
     If set to a positive integer, the GUI will display an HTTP link to the IP
     address which is currently used for synchronization.  Only the TCP port is
     exchanged for the value specified here.  Note that any port forwarding or
     firewall settings need to be done manually and the link will probably not
     work for link-local IPv6 addresses because of modern browser limitations.
 
+.. option:: device.untrusted
+
+    This boolean value marks a particular device as untrusted, which disallows
+    ever sharing any unencrypted data with it.  Every folder shared with that
+    device then needs an encryption password set.  Refer to the detailed
```

Comment:
I think you can also share receive encrypted folders with such devices, can't you? Those don't need to have a password set on the device that shares them.

Reply:
I don't know. I followed what was described on the [dedicated page](https://docs.syncthing.net/users/untrusted.html#gui):

> If you never want to sync any unencrypted data to a particular device, you can mark it as untrusted in the advanced tab of the device edit dialog. This will prevent a connection to that device if you forget to set a password on any folder shared with it.

Reply:
The intention is definitely that you can share an already encrypted folder with an untrusted device. I haven't thought about that when implementing though, so not sure it works. I'd consider it a bug though if it didn't.

Reply:
I've just tested this. I can confirm you can share an already encrypted folder with an untrusted device. The only caveat is that the folder type must be manually adjusted to "receive encrypted" on the untrusted side, but other than that, everything seems to be working as expected.

Reply:
So, any suggestion what changes are needed in the text?

Reply:
I committed a possible solution.  It also documents the folder.type = `receiveencrypted`, which was previously missed.